src/server/bg_services/log_replication_scanner.js

```diff
@@ -141,6 +156,32 @@ class LogReplicationScanner {
         }
     }
 
+    async process_conflict_candidate(src_bucket, dst_bucket, candidate) {
+        // If the item is present in the source bucket but not in the destination bucket
+        // then it is 'copy' action
```

Comment:
we should handle also the following scenario - if (candidate.src_object_info && candidate.dst_object_info)
when the source object is newer than the destination

Reply:
Actually I knowingly left out this scenario in current as well as previous PR as it is handled already right now by the scan based replication.

Maybe we can bring this change in another PR as it is not specific to the fix? WDYT?